pages/api/upload-user-to-airtable.ts

```diff
@@ -0,0 +1,58 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import Airtable from 'airtable'
+import * as R from 'ramda'
+
+//eslint-disable-next-line
+import { prisma } from '../../prisma/generated/ts'
+
+Airtable.configure({
+  endpointUrl: 'https://api.airtable.com',
+  apiKey: process.env.AIRTABLE_API_KEY,
+})
+
+const base = Airtable.base(process.env.AIRTABLE_BASE)
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { ages, postCode, project, consentGDPR = true, user } = req.body
```

Comment:
maybe it would be better to default to false for the consentGDPR value?

Reply:
or even to throw an error if this value isn't defined or is false? though I don't know how we'd handle that. I guess it should never reach that point anyway. .

Reply:
Annoyingly, the consent context wasn't persisting because Auth0 restarts the page. So, I've defaulted this to true for now because to get to this part of the process one needs to have consented. I've kept in the context stuff in the hope that we can work out how to pass this on in future — potentially putting into storage?